lightning/src/routing/scoring.rs

```diff
@@ -4139,21 +4139,24 @@ mod tests {
 			short_channel_id: 42,
 		});
 
-		// Apply some update to set the last-update time to now
-		scorer.payment_path_failed(&payment_path_for_amount(1000), 42, Duration::ZERO);
+		// Initialize the state for channel 42
+		scorer.payment_path_failed(&payment_path_for_amount(500), 42, Duration::ZERO);
+
+		// Apply an update to set the last-update time to 1 second
+		scorer.payment_path_failed(&payment_path_for_amount(490), 42, Duration::from_secs(1));
```

Comment:
If we don't update the bounds, didn't we still learn that the bounds are still valid? And maybe the data point time needs to be updated in that case?